models/_sector/nft/trades/nft_ethereum_trades_beta.sql

```diff
@@ -0,0 +1,31 @@
+{{ config(
+    schema = 'nft_ethereum',
+    alias ='trades_beta',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['project','project_version','tx_hash','sub_tx_trade_id']
+    )
+}}
+-- (project, project_version, model)
+{% set base_models = [
+     ('archipelago',    'v1',   ref('archipelago_ethereum_base_trades'))
+    ,('superrare',    'v1',   ref('superrare_ethereum_base_trades'))
+    ,('foundation',    'v1',   ref('foundation_ethereum_base_trades'))
+] %}
+
+
+-- macros/models/sector/nft
+{{
+    enrich_trades(
+        blockchain='ethereum',
+        models=base_models,
+        transactions_model=source('ethereum','transactions'),
+        tokens_nft_model=ref('tokens_ethereum_nft'),
+        tokens_erc20_model=ref('tokens_ethereum_erc20'),
+        prices_model=ref('prices_usd_forward_fill'),
```

Comment:
is using `forward_fill` a new approach?

Reply:
I'd say it's preferred over the regular `prices.usd` as it enables more recent trades to also have a `amount_usd` value.
It won't be 100% accurate but will be on following runs once price.usd has catched up. 

I know Hildobby has been using it in his marketplace models so I also did not want to remove that here.